hedera-node/src/test/java/com/hedera/test/utils/TxnUtils.java

```diff
@@ -84,6 +85,28 @@ public static TransferList withAdjustments(
 		);
 	}
 
+	public static TokenTransferList withNFTAdjustments(
```

Comment:
nit: usually, we do not specify `NFT` all upper case. Everywhere in the code you can see it is `Nft`